ironfish-cli/src/commands/accounts/transactions.ts

```diff
@@ -0,0 +1,113 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { oreToIron } from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class TransactionsCommand extends IronfishCommand {
+  static description = `Display the account transactions`
+
+  static flags = {
+    ...RemoteFlags,
+    account: Flags.string({
+      char: 'a',
+      description: 'account transactions',
+    }),
+    hash: Flags.string({
+      char: 't',
+      description: 'details of transaction hash',
+    }),
+  }
+
+  async start(): Promise<void> {
+    const { flags } = await this.parse(TransactionsCommand)
+    const account = flags.account?.trim()
+    const hash = flags.hash?.trim()
+
+    if (hash) {
+      await this.getTransaction(account, hash)
+    } else {
+      await this.getTransactionsFor(account)
+    }
+  }
+
+  async getTransaction(account: string | undefined, hash: string): Promise<void> {
+    const client = await this.sdk.connectRpc()
+
+    const response = await client.getAccountTransaction({ account, hash })
+
+    const {
+      account: accountResponse,
+      transactionHash,
+      transactionInfo,
+      transactionNotes,
+    } = response.content
+
+    this.log(`Account: ${accountResponse}`)
+
+    if (transactionInfo !== null) {
+      this.log(
+        `Transaction: ${transactionHash}, Status: ${transactionInfo.status}, Miner Fee: ${
```

Comment:
What do you think about making each of these on a new line? I think it makes it a little easier to parse the info.
```
Transaction: fdaf29274587c8f58ac6214f42642fd610a222cea8e5e39ff4e11d58d379c194
Status: completed
Miner Fee: x
Fee ($ORE): 1
Spends: 1

```

Reply:
update to this